nilearn/datasets/utils.py

```diff
@@ -617,11 +617,12 @@ def _fetch_files(data_dir, files, resume=True, mock=False, verbose=1):
         Unique dataset name
 
     files: list of (string, string, dict)
-        List of files and their corresponding url. The dictionary contains
-        options regarding the files. Options supported are 'uncompress' to
-        indicates that the file is an archive, 'md5sum' to check the md5 sum of
-        the file and 'move' if renaming the file or moving it to a subfolder is
-        needed.
+        List of files and their corresponding url with dictionary that contains
+        options regarding the files. Ex. (file_name, url, opt). If a file_name
+        is not found in data_dir, as in data_dir/file_name the download will
+        be cancelled at the end. Options supported are 'uncompress' to indicate
```

Comment:
If a filename is not found, the download is immediately cancelled (with your sentence, I have the impression that we download the rest of the files before failing). You should also state that the downloaded files are erased (I'll add an option to override this behavior later).